### ISSUE TITLE
Use of "_include" parameter is not a feature on the PaaS (Cosmos DB) version

### ISSUE THREAD

**Issue by MJ422** [microsoft/fhir-server#1226]
**Describe the bug**
Hello all,
I am trying to figure out how we can support the use of the "_include" parameter in a FHIR API search. We are currently utilizing the PaaS version, which is based on the Cosmos DB framework.

We understand that this is not a supported feature of this version of the service, according to the features matrix, however we would like to discuss any possible way to modify the code for Azure API for FHIR to add that functionality. 

**To Reproduce**
1. Search FHIR API for resource with _include parameter
2. Find that this operation is not supported by Cosmos 

**Expected behavior**
To search resources with the ?_include parameter, and receive Bundle with referenced resources appended in the entry array

**Actual behavior**
"Operation not supported"

TL;DR: How can we use the _include parameter operation on the Azure API for FHIR PaaS version connected to Cosmos? If we want to add that functionality ourselves, where in the API for FHIR codebase can we modify such that we can support this param?

**Comment by johnstairs** [comment]
Thanks for the feedback. We are looking into supporting this on Cosmos DB.

**Comment by CaitlinV39** [comment]
@MJ422 right now we don't have plans to implement this on Cosmos DB. We are going to close this as won't fix for now.
